### PR TITLE
DOC-7526: Communicate a channel’s focus and scope page needs animated…

### LINKED ISSUE #7526: Communicate a channel’s focus and scope page needs animated GIFs for mobile app workflows
## Context

To help visual learners learn how to perform tasks using the Mattermost mobile app, add visuals and short animated GIFs on the [Communicate a channel’s focus and scope](https://docs.mattermost.com/collaborate/channel-header-purpose.html) page.

## Docs Request

- Add appropriate visuals/GIFs for all the steps in the `Channel purpose` section under the mobile app tab.
- Add appropriate visuals/GIFs for all the steps in the `Channel header` section under the mobile app tab.

## Additional Notes

- Use the latest Mattermost mobile app and the standard Mattermost Denim theme to capture all workflows.
- Avoid drawing lines, arrows, boxes, or callouts on images.
- Don't forget to include ALT Text descriptions for all new image assets.
- Use existing docs pages with images as a starting point for code guidance. The Docs team can help provide RST syntax support for publication.
- Save static image files in PNG format within the docs repository's `/images` directory.

# QA Validation Criteria

In a generated docs preview:
- Visiting the docs page results in appropriate new mobile-specific screenshots in each subsection from the **Mobile** tab.

### PR DESCRIPTION
**Pull request by Rutam21**
### Summary

This PR adds visual workflows for the Mattermost mobile app tab on the [Communicate a channel’s focus and scope](https://docs.mattermost.com/collaborate/channel-header-purpose.html) page

### Ticket Link

Fixes #7526 

- [x] This PR follows Mattermost Contributing Guidelines.

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

@lindy65 Please add this to my queue for QA Testing once the reviews are complete. Thank you.

**Comment by lindy65** [comment]
Assigned to you for testing @Rutam21 👍

File: source/collaborate/channel-header-purpose.rst [hunk 1]
```diff
@@ -41,9 +41,34 @@ A channel purpose can be up to 250 characters in length, and is often used to he
 .. tab:: Mobile
 
   1. Tap the channel you want to edit.
-  2. Long-press on the channel purpose.
-  3. Tap **Copy purpose text**.
-  4. Tap the purpose field to paste the copied text.
+
```
Comment (cwarnermm) [inline source/collaborate/channel-header-purpose.rst:44]: @Rutam21 - The new code you've added is intended to the right by a single character, and this is enough to cause display issues (see the generated preview for details).

Removing the extra space on each code line should resolve the display issues on this page (both sections).

Comment (Rutam21) [reply source/collaborate/channel-header-purpose.rst:44]: I have already updated it. Please re-check. Thanks.

Comment (cwarnermm) [reply source/collaborate/channel-header-purpose.rst:44]: Looks like you were applying these updates as I was commenting on them! Great minds think alike! :)

Comment (Rutam21) [reply source/collaborate/channel-header-purpose.rst:44]: Yeah, indeed. I was also checking the preview after it was generated.

File: source/collaborate/channel-header-purpose.rst [hunk 2]
```diff
@@ -41,9 +41,34 @@ A channel purpose can be up to 250 characters in length, and is often used to he
...
+  .. image:: ../images/mobile-select-a-channel.jpg
+    :alt: Select a channel that you want to edit.
+    :scale: 30
+
+  2. Tap the **More** |more-icon-vertical| icon located in the top right corner of the app.
+
+  .. image:: ../images/mobile-select-more-options-for-a-channel.jpg
+    :alt: Tap on More options to access available options for the channel.
+    :scale: 30
```
Comment (cwarnermm) [inline source/collaborate/channel-header-purpose.rst:53]:
```suggestion
    .. image:: ../images/mobile-select-more-options-for-a-channel.jpg
      :alt: Tap on More options to access available options for the channel.
      :scale: 30
```

File: source/collaborate/channel-header-purpose.rst [hunk 3]
```diff
@@ -41,9 +41,34 @@ A channel purpose can be up to 250 characters in length, and is often used to he
...
+
+  .. image:: ../images/mobile-select-a-channel.jpg
+    :alt: Select a channel that you want to edit.
+    :scale: 30
+
+  2. Tap the **More** |more-icon-vertical| icon located in the top right corner of the app.
+
+    .. image:: ../images/mobile-select-more-options-for-a-channel.jpg
+      :alt: Tap on More options to access available options for the channel.
+      :scale: 30
```
Comment (cwarnermm) [inline source/collaborate/channel-header-purpose.rst:53]:
```suggestion
    .. image:: ../images/mobile-select-more-options-for-a-channel.jpg
        :alt: Tap on More options to access available options for the channel.
        :scale: 30
```

File: source/collaborate/channel-header-purpose.rst (hunk 1 above)
Comment (cwarnermm) [reply source/collaborate/channel-header-purpose.rst:44]: Latest preview continues to show the images as clickable links. 

What's baffling is that mobile #5 in both sections displays as intended. I have yet to confirm why it's working and the rest aren't. I may need to pull down this fork locally and investigate further.

**Comment by Rutam21** [comment]
@cwarnermm I got the bug. This PR uses the image files added in this PR: https://github.com/mattermost/docs/pull/7525. Hence, those image files are missing from this preview branch. This should resolve itself after we merge that PR and try building another preview.

**Comment by cwarnermm** [comment]
Thanks, @Rutam21 for that heads-up! Splitting image files across PRs would definitely net poor results in previews. I've merged the related PR. Once published, I'll confirm it's working as expected, then come back to this PR. If previews still aren't working, I'll either pull the fork down locally OR may merge then fix.

**Comment by Rutam21** [comment]
@cwarnermm Points noted. I will keep this in mind while raising the next PRs. However, the preview still seems to be broken but the changes work after the merge. Please find a screenshot of the local docs build attached.

Screenshot 2024-11-01 022136

**Comment by cwarnermm** [comment]
The shorter path through on this PR is to merge it and investigate/fix it in production. So that's what I'm going to do.

**Pull request merged**